clean-ci-resources.sh

```diff
@@ -58,6 +58,14 @@ openstack container list -f value -c Name \
 	| xargs --verbose --no-run-if-empty openstack container delete -r
 
 for resource in 'volume snapshot' 'volume' 'floating ip'; do
+	if [[ ${resource} == 'volume' ]]; then
+	  for r in $(./stale -q "$resource"); do
+	    status=$(openstack "${resource}" show -c status -f value "${r}")
+	    if [[ ${status} == 'reserved' ]]; then
```

Comment:
we need to support more states, see https://docs.openstack.org/api-ref/block-storage/v3/index.html?expanded=delete-a-volume-detail#delete-a-volume